src/main/webapp/app/overview/course-lectures/course-lecture-details.component.html

```diff
@@ -52,9 +52,9 @@ <h3 jhiTranslate="artemisApp.courseOverview.lectureDetails.description"></h3>
                     }
                     <!-- LECTURE UNITS START-->
                     @if (lecture.lectureUnits) {
-                        <div class="row mb-2 mt-2 align-items-baseline">
+                        <div class="row mb-2 mt-2 align-items-center">
                             <div class="col-auto">
-                                <h3 jhiTranslate="artemisApp.courseOverview.lectureDetails.lectureUnits"></h3>
+                                <h3 class="mb-0" jhiTranslate="artemisApp.courseOverview.lectureDetails.lectureUnits"></h3>
```

Comment:
Do you add the `mb-0` to overwrite some other bottom-margin property?

Reply:
`h3` has no mb specified, so I thought I would be overwriting the default margin of 1em to 0.
I could also instead of moving down `Lecture Units`, move `Download Complete PDF` up.
If there is another prefered way, I would be happy to update that.